datasets/librispeech_asr/librispeech_asr.py

```diff
@@ -69,6 +75,14 @@ def map_to_array(batch):
         "dev": _DL_URL + "dev-other.tar.gz",
         "train.500": _DL_URL + "train-other-500.tar.gz",
     },
+    "all": {
+        "dev": {retrieve_subset(u): _DL_URL + u for u in ["dev-clean.tar.gz", "dev-other.tar.gz"]},
```

Comment:
I think it is more standard to split this into dev-clean and dev-other because you usually always treat and report them separately.

Reply:
sounds good, we can make that change